README.md

```diff
@@ -91,9 +91,27 @@ Development
 
 [![Build Status](https://travis-ci.org/hickford/MechanicalSoup.svg?branch=master)](https://travis-ci.org/hickford/MechanicalSoup)
 
-### Tests
+You can develop against multiple versions of Python using virtualenv:
+
+    python3 -m venv .virtual-py3 && source .virtual-py3/bin/activate
```

Comment:
It was on a Debian oldstable (I don't have it at hand, but should be 3.4.2 according to Debian's website). I tested again on another machine and your version works.

However, you're suggesting `python3 -m venv` for Python3 and then `virtualenv -p python2` for Python2. I think it'd be better to use the same syntax (no preference on which one on my side) for both.

Also, the link you just gave is a good one, it would be nice to include it in your explanation (for example as a hyperlink on the first occurence of "virtualenv").

Thanks,